src/api.js

```diff
@@ -23,7 +23,7 @@ export const getConferenceById = async id => {
 export const createNewConference = async conference => {
   try {
     const response = await axios.post(
-      "http://localhost:8080/conferences",
+      `http://localhost:8080/conferences`,
```

Comment:
Is there a reason this string is not using normal quotes?

Reply:
Hi Ross, do you mean the post request should be in back ticks? (thats how it looks in our version) or are you suggesting we change to ""??

Reply:
Normally you only use back ticks for when you are concatenating variables and strings together. So if it's only a string I'd use normal ""

Reply:
for context thats using template literal strings 👍